native-sql-engine/cpp/src/CMakeLists.txt

```diff
@@ -214,45 +214,45 @@ macro(build_arrow STATIC_ARROW)
   else()
 
     # Copy Arrow Shared Library to releases directory for package jar
-    ExternalProject_Add_Step(arrow_ep copy_arrow_binary
+    #ExternalProject_Add_Step(arrow_ep copy_arrow_binary
```

Comment:
remove this?

Reply:
Yes, I commit a new one to remove the marked code.